bloom-housing/bloom
# fix: community type disclaimer frontend

This PR addresses #4476 #4477 


## Description

Adds optional disclaimer to application. Both partners and public site.

## How Can This Be Tested/Reviewed?

In partners site in reserved community type there is new section that allows to specify if that section should appear. If it's yes it should require both fields (title and disclaimer). Make sure it works fine for editing, adding new listing, when other fields throwing an error etc. (as there was issues with that in meantime).
when "yes" and both fields filled it should appear as first step of application (right after language select). Make sure it works also without it, and form works properly (like navigation back etc)

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
